controllers/tasks.js

```diff
@@ -233,10 +233,8 @@ const updateTaskStatus = async (req, res, next) => {
     if (task.taskData.status === TASK_STATUS.VERIFIED || req.body.status === TASK_STATUS.MERGED)
       return res.boom.forbidden("Status cannot be updated. Please contact admin.");
 
-    if (task.taskData.status === TASK_STATUS.COMPLETED && req.body.percentCompleted < 100) {
-      if (req.body.status === TASK_STATUS.COMPLETED || !req.body.status) {
-        return res.boom.badRequest("Task percentCompleted can't updated as status is COMPLETED");
-      }
+    if (task.taskData.status === TASK_STATUS.COMPLETED) {
+      return res.boom.badRequest("Task details can't be updated as it is marked as COMPLETED");
```

Comment:
we should only check this if the request body has status. Maybe we would like to change the `completionAward` or `startDate` or some other parameter. if in the request body we receive new status and the current status is completed then we shouldn't allow. Also please check with @ankushdharkar once that whether is there any state after completed. Maybe `released` might come after completed. In that case we must allow `released`. just check with Ankush once.

Reply:
changes reverted as discussed